mineral-ui/mineral-ui
# [Button] [MenuItem] Style updates

### Description

- Update Button spacing/layout and colors to match [spec](https://ca.invisionapp.com/d/main#/console/11070131/250301080/preview)
- Update MenuItem spacing/layout and colors to match [spec](https://ca.invisionapp.com/d/main#/console/11070131/250301075/preview)
    - Including revised variant icons
- Update `mineralTheme` to accomplish the above
- Update snapshots

### Motivation and context

Closes #341 

### Screenshots, videos, or demo, if appropriate


https://341-button-menu-item-updates--mineral-ui.netlify.com/

### How to test


1. `cd mineral-ui && npm start`
2. Navigate to Button and confirm visual accuracy
3. Navigate to MenuItem and confirm visual accuracy

### Types of changes

- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

Updated Button and MenuItem components

Breaking: Removed some theme variables in Button and MenuItem

### Checklist

* [x] Renders and functions properly in [all supported browsers](https://github.com/mineral-ui/mineral-ui#browser-support) **See below**



The Button text sits a bit low in Edge, and a lot low in IE 11.
